### ISSUE TITLE
Sign PSBT Transaction

### ISSUE THREAD

**Issue by zebpayamarthakur** [bcoin-org/bledger#29]
Hi, how can I sign PSBT Transaction using ledger ?

**Comment by pinheadmz** [comment]
I don't think this is currently supported. PSBT is a work-in-progress by bcoin contributor @joemphilips who might be able to chime in here ;-)

**Comment by tynes** [comment]
See this from the bitcoin-dev mailing list:
https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-August/014843.html

And see this discussion:
https://github.com/trezor/python-trezor/issues/183

**Comment by zebpayamarthakur** [comment]
@nodar-chkuaselidze When Bcoin & Bledger will start supporting PSBT ?

**Comment by nodar-chkuaselidze** [comment]
Hey, there's Pull Request pending on bcoin, after it's merged and published with next version of bcoin, Bledger as well as other projects will start supporting it.

**Comment by zebpayamarthakur** [comment]
Ok. Great.

**Comment by jonathancross** [comment]
> next version of bcoin...

Does this mean PSBT support has landed in [v0.2.0](https://github.com/bcoin-org/bledger/releases/tag/v0.2.0)?

**Comment by nodar-chkuaselidze** [comment]
No unfortunately not yet.
https://github.com/bcoin-org/bledger/blob/v0.2.0/CHANGELOG.md this is the changelog what was added to bledger.

PSBT has not been merged in bcoin yet, but I think it will for next bcoin release.

**Comment by jonathancross** [comment]
Did PSBT make it into v0.2.1 ?
